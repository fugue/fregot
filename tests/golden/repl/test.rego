package fregot.tests.repl.test

test_foo {
  1 + 1 > 1
}
