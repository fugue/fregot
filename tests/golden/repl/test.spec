{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${SPEC_NAME}.stderr"},
        {"stdout": "${SPEC_NAME}.stdout"}
    ],
    "stdin": [
        ":load test.rego",
        ":test"
    ]
}
