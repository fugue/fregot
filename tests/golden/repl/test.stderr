F u g u e   R E G O   T o o l k i t
fregot v0.7.4 repl - use :help for usage info
Loading test.rego...
Loaded package fregot.tests.repl.test
